packages/shared/src/db/queries.ts

```diff
@@ -1295,7 +1295,13 @@ export const insertChannelPerms = createWriteQuery(
           set: conflictUpdateSetAll($channelWriters),
         });
     }
+  },
+  ['channelWriters', 'channels']
+);
 
+export const insertChannelOrder = createWriteQuery(
+  'insertChannelOrder',
+  async (channelsInit: ChannelInit[], ctx: QueryCtx) => {
```

Comment:
narrowing the type here would be nice to avoid future conflation

```suggestion
  async (channelsInit: Array<{ order: string[] }>, ctx: QueryCtx) => {
```